homeassistant/components/blink/services.yaml

```diff
@@ -26,3 +26,11 @@ send_pin:
     pin:
       description: Pin received from blink. Leave empty if you only received a verification email.
       example: "abc123"
+
+record:
+  description: Request a camera to record a new video.
+  fields:
+    entity_id:
```

Comment:
Please update the service description according to our latest model. See
https://developers.home-assistant.io/docs/dev_101_services#service-descriptions
and
https://www.home-assistant.io/docs/blueprint/selectors#target-selector